cmd/flux/cmd/test.go

```diff
@@ -0,0 +1,151 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/ast/edit"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/execute/table"
+	"github.com/influxdata/flux/fluxinit"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/parser"
+	"github.com/influxdata/flux/runtime"
+	"github.com/spf13/cobra"
+)
+
+type Test struct {
+	ast *ast.Package
+	Err error
```

Comment:
API nitpick: Can you make this `err` and make it accessible from an `Err() error` getter?